Makefile

```diff
@@ -87,13 +88,13 @@ test:	rclone test_all
 
 # Quick test
 quicktest:
-	RCLONE_CONFIG="/notfound" go test $(BUILDTAGS) ./...
+	RCLONE_CONFIG="/notfound" go test -v $(LDFLAGS) $(BUILDTAGS) ./cmd/gitannex/...
```

Comment:
Whoops, I didn't mean to restrict tests to `./cmd/gitannex/...`. I'll fix it next time I work on this PR.